src/external_command_helper.c

```diff
@@ -20,6 +24,11 @@
 #if _WIN32
 #define popen _popen
 #define pclose _pclose
+#define POPEN_MODE "rt"
+#define PCLOSE_RETURN_SHIFT 0
+#else
+#define POPEN_MODE "r"
+#define PCLOSE_RETURN_SHIFT 8
 #endif

```

Comment:
I made a task to put popen/pclose in the PAL, but this at least gets things working for now - Task 9124184: popen/pclose to PAL